src/analysis/complex/upper_half_plane/functions_bounded_at_inf.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck, David Loeffler
+-/
+
+import algebra.module.submodule.basic
+import analysis.complex.upper_half_plane.basic
+
+/-!
+# Bounded at infinity
+
+For complex valued functions on the upper half plane, this file defines the notion of bounded at
+infinity and zero at infinity. Both of which are relevant for defining modular forms.
+-/
+
+open complex
+
+open_locale topological_space upper_half_plane
+
+namespace upper_half_plane
+
+/--Filter for approaching `i∞`-/
+def at_I_inf := filter.at_top.comap upper_half_plane.im
```

Comment:
We use `inf` for the lattice meet operation and `at_top`/`infty` for "tends to infinity"